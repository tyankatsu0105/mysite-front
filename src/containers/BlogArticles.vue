<template>
	<div class="BlogArticles">
		<blog-article
			v-for="edge in page.allWordPressPost.edges"
			:key="edge.node.id"
			heading-tag="h2"
			:article="edge.node"
		/>
	</div>
</template>

<script>
import BlogArticle from "@/components/BlogArticle";
export default {
  components: {
    BlogArticle
  },
  props: {
    page: {
      type: Object,
      default: () => {}
    }
  }
};
</script>

<style scoped lang='scss'>
.BlogArticles {
  display: grid;
  grid-template-columns: 1fr 1fr 1fr;
  grid-auto-rows: auto;
  grid-gap: 30px;
  justify-items: center;

  @include mq-xs {
    grid-template: 1fr / 1fr;
  }
}
</style>
