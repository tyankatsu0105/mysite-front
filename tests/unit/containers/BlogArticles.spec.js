import { shallowMount } from "@vue/test-utils";
import BlogArticles from "@/containers/BlogArticles.vue";

describe("BlogArticles", () => {
  it("render", () => {
    const wrapper = shallowMount(BlogArticles);

    expect(wrapper.html()).toMatchSnapshot();
  });
});
