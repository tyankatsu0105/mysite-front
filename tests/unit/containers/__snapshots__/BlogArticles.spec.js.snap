// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BlogArticles render 1`] = `
<div class="BlogArticles">
  <blog-article-stub headingtag="h2" article="[object Object]"></blog-article-stub>
  <blog-article-stub headingtag="h2" article="[object Object]"></blog-article-stub>
</div>
`;
